packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/layout-customization/index.hbs

```diff
@@ -2,17 +2,58 @@
   @header="Layout Customization"
   @isComplete={{@isComplete}}
   data-test-layout-customization-is-complete={{@isComplete}}
+  class="layout-customization"
 >
+<section class="layout-customization__body">
+  <header class="layout-customization__body-header">
+    Customize the layout of your prepaid card
+  </header>
+  {{#if @isComplete}}
+    <CardPay::IssuePrepaidCardWorkflow::LayoutCustomization::DisplayView
+      @issuerName={{this.issuerName}}
+      @headerBackground={{this.headerBackground}}
+      @headerTheme={{this.headerTheme}}
+    />
+  {{else}}
+    <CardPay::IssuePrepaidCardWorkflow::LayoutCustomization::FormView
+      @issuerName={{this.issuerName}}
+      @colorOptions={{this.colorOptions}}
+      @themeOptions={{this.themeOptions}}
+      @headerBackground={{this.headerBackground}}
+      @updateHeaderBackground={{this.updateHeaderBackground}}
+      @headerTheme={{this.headerTheme}}
+      @updateHeaderTheme={{this.updateHeaderTheme}}
+    />
+  {{/if}}
+
+  <div class="layout-customization__disclaimer">
+    * The issuer name will be published on the blockchain and cannot be deleted. Please don’t include any personally identifiable information that you want to keep private.
```

Comment:
I don't think we're going to publish the issuer name to the blockchain.

Maybe better to say: " * The issuer name will be visible to others. Please don’t include any personally identifiable information that you want to keep private."

Reply:
How are we going to store the issuer name prior to persistence?